shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -2695,6 +2695,9 @@ private void updateRecursively(
         if (globalTransform == null) {
           globalTransform = new float[16];
         }
+        if (transform == null) {
```

Comment:
I imagine you still need another condition to initialize it  so it won't crash in release code.

I did try to compile a custom engine in release mode, but it looks like the assertion will still crash the app. @GaryQian Do you know if the assertionError will be ignored in release mode?

Reply:
You meant the exception will be thrown only on debug mode, right? Shall BuildConfig.DEBUG work?

Reply:
yeah I think BuildConfig.DEBUG is probably the best way to do this

Reply:
will reedit it.